Unix/installbuilder/conf/omiserver.conf

```diff
@@ -33,31 +35,49 @@ httpsport=0
 #NtlmCredsFile=/etc/opt/omi/.creds/ntlm
 
 ##
-## <NICKNAME> -- set the value of nickname.
-##
-#prefix=PATH
-#libdir=PATH
-#bindir=PATH
-#localstatedir=PATH
-#sysconfdir=PATH
-#providerdir=PATH
-#certsdir=PATH
-#datadir=PATH
-rundir=/var/opt/omi/run
-#logdir=PATH
-#schemadir=PATH
-#schemafile=PATH
-pidfile=/var/opt/omi/run/omiserver.pid
-logfile=/var/opt/omi/log/omiserver.log
-registerdir=/etc/opt/omi/conf/omiregister
-pemfile=/etc/opt/omi/ssl/omi.pem
-keyfile=/etc/opt/omi/ssl/omikey.pem
-#agentprogram=PATH
-#serverprogram=PATH
-#includedir=PATH
-configfile=/etc/opt/omi/conf/omiserver.conf
+## This is the setting for omi related directory and file path, the following values are default values, and we don't support modifing the path.
+##   Please don't modify any path value, otherwise the omi will not work!
+##
```

Comment:
How about changing 38-39 to something like:

```
## Settings for omi-related directory and file paths.
##   Do not change any values below. Otherwise, OMI won't function properly!
```

Reply:
But wait: If these are all DEFAULT values, can't you just get rid of them entirely?

If everything is commented out (default values), and we can't change them due to how we build source code, can't the entire segment be removed from this file?

Please validate and resubmit your review, thanks.

Reply:
Yes, we can remove the default values, I will update them later, thanks.

Reply:
Removed commented default values section.